src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,104 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private static User testUser;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        Storage.people.clear();
+        testUser = new User();
+        testUser.setPassword("valid_password");
+        testUser.setLogin("valid_login");
+        testUser.setAge(21);
+    }
+
+    @Test
+    void register_nullValue_NotOk() {
+        assertThrows(RuntimeException.class, () -> registrationService.register(null));
+    }
+
+    @Test
+    void register_userWithNullAge_NotOk() {
+        testUser.setAge(null);
+        assertThrows(RuntimeException.class, () -> registrationService.register(testUser),
+                "Null age shouldn't be accepted.");
+    }
+
+    @Test
+    void register_userWithNullLogin_NotOk() {
+        testUser.setLogin(null);
+        assertThrows(RuntimeException.class, () -> registrationService.register(testUser),
+                "Null login shouldn't be accepted.");
```

Comment:
let's remove messages from `assertThrows`, it's redundant because you have already written messages in `RegistrationServiceImpl`